docs/api.md

```diff
@@ -2214,10 +2611,6 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 + code: `501.1` (string, required)
 + message: `The requested feature $unsupported is not supported by this server.` (string)
 
-## App User (Actor)
-+ createdBy: `42` (number, required) - The ID of the `Actor` that created this `App User`.
-+ token: `d1!E2GVHgpr4h9bpxxtqUJ7EVJ1Q$Dusm2RBXg8XyVJMCBCbvyE8cGacxUx3bcUT` (string, optional) - If present, this is the Token that can be used to authenticate a request as this `App User`. If not present, this `App User`'s access has been revoked.
-
 ## Form (object)
```

Comment:
`null` now specifically means "there is no published/active form," right? If the user uploads an XForm without a version, does that always become empty string in the database, or is it `null`?

Reply:
if you upload a form with no version, it will be defaulted to blank string. so, yes, if `version` is `null` or not provided then there is no version. probably the sha or one of the other fields is a more reassuring value to look at. i can clarify this a little.